_maps/map_files/TgStation/tgstation.2.1.3.dmm

```diff
@@ -15467,7 +15467,8 @@
 "aFA" = (
 /obj/machinery/atmospherics/pipe/simple/supply/hidden,
 /obj/machinery/computer/pod/old{
-	icon = 'icons/obj/airlock_machines.dmi';
+	density = 0;
+	 icon = 'icons/obj/airlock_machines.dmi';

```

Comment:
Did you use spaces instead of tabs or is github just displaying these two lines strangely?